core/templates/dev/head/domain/statistics/LearnerActionObjectFactory.js

```diff
@@ -48,9 +61,20 @@ oppia.factory('LearnerActionObjectFactory', [function() {
    */
   LearnerAction.createFromBackendDict = function(learnerActionBackendDict) {
     return new LearnerAction(
-      learnerActionBackendDict.actionType,
-      learnerActionBackendDict.actionCustomizationArgs,
-      learnerActionBackendDict.schemaVersion);
+      learnerActionBackendDict.action_type,
+      learnerActionBackendDict.action_customization_args,
+      learnerActionBackendDict.schema_version);
+  };
+
+  /**
+   * @returns {LearnerActionBackendDict}
+   */
+  LearnerAction.prototype.convertToBackendDict = function() {
```

Comment:
Hmm, `convertToBackendDict` makes it sound like the object is having its internals changed to become a backend dict. I think `toBackendDict` is clearer.

Reply:
Done.